### PR TITLE
setup rendering pipeline for echarts static pie

### PR DESCRIPTION
**Pull request by EmmadUsmani**
Part of https://github.com/metabase/metabase/issues/33281

### Description

Sets up the static viz rendering pipeline for our new static pie chart implementation using echarts. Currently it uses some mock values that will be replaced by the actual implementation in later PRs in this stack. It also currently adds a dynamic pie chart visualization type to the client, but this will also be removed later in the stack before it is all merged to master.

### How to verify

1. Create a question
2. Set visualization type to `Pie 2`
3. Add to a dashboard
4. Send in an email subscription
5. Confirm that a placeholder chart is rendered without any errors

### Demo

^ output from subscription email

[Screen Recording 2023-10-16 at 2.00.12 PM.mov](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/9ULpbAzGBtvQRyZEpGvu/92243e9e-4481-4edd-a6b6-3abb6530db63/Screen%20Recording%202023-10-16%20at%202.00.12%20PM.mov)

### Checklist

- [ ] Tests have been added/updated to cover changes in this PR

_TODO: Need to update `js_svg_test.clj`_

### CODE REVIEW

**Comment by EmmadUsmani** [comment]
Current dependencies on/for this PR:
* master
  * **PR #34606** <a href="https://app.graphite.dev/github/pr/metabase/metabase/34606" target="_blank">Graphite</a> 
    * **PR #34717** <a href="https://app.graphite.dev/github/pr/metabase/metabase/34717" target="_blank">Graphite</a>  👈
    * **PR #34703** <a href="https://app.graphite.dev/github/pr/metabase/metabase/34703" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/metabase/metabase/34717?utm_source=stack-comment).

File: src/metabase/pulse/render.clj [hunk 1]
```diff
@@ -100,6 +100,7 @@
            :area
            :bar
            :combo
+           :pie2
```
Comment (EmmadUsmani) [inline src/metabase/pulse/render.clj:103]: This name will also change later in the stack when we remove the dynamic chart used for testing

File: src/metabase/pulse/render/body.clj [hunk 2]
```diff
@@ -868,6 +868,19 @@
       [:img {:style (style/style {:display :block :width :100%})
              :src   (:image-src image-bundle)}]]}))
 
+(s/defmethod render :pie2 :- common/RenderedPulseCard
```
Comment (EmmadUsmani) [inline src/metabase/pulse/render/body.clj:871]:
```suggestion
(s/defmethod render :pie :- common/RenderedPulseCard
```

**Review by alxnddr — APPROVED** [review]

File: dev/src/dev/render_png.clj [hunk 3]
```diff
@@ -87,9 +88,7 @@
 
 (comment
   (render-card-to-png 1)
-
-  (let [{:keys [content]} (render-pulse-card 1)]
-    (open-hiccup-as-html content))
+  (render-card-to-png 2700)
```
Comment (camsaul) [inline dev/src/dev/render_png.clj:91]: I don't think this change is adding anything, the old version was there to help you see how to use this stuff.

Comment (EmmadUsmani) [reply dev/src/dev/render_png.clj:91]: Sorry, this was a really old PR I should have closed. The new version is [here]([url](https://github.com/metabase/metabase/pull/43555)) (still a wip progress)

**Pull request closed without merging**